flutter-for-react-native/index.md

```diff
@@ -0,0 +1,2538 @@
+ the edge of a `Scaffold` to show navigation links in an application. You can
+  provide a [`Button`](https://docs.flutter.io/flutter/material/RaisedButton-class.html), a [`Text`](https://docs.flutter.io/flutter/widgets/Text-class.html) widget, or a list of items to display as the child to the
+  `Drawer` widget. In the following example, the  [`ListTile`](https://docs.flutter.io/flutter/material/ListTile-class.html)widget provides the
+  navigation on tap.
+
+<!-- skip -->
+```Dart
+// Flutter
+new Drawer(
+  child:new ListTile(
+    leading: new Icon(Icons.change_history),
+    title: new Text('Screen2'),
+    onTap: () {
+      Navigator.of(context).pushNamed("/b");
+    },
+  ),
+  elevation: 20.0,
+),
+```
+
+The `Scaffold` widget also includes an `AppBar` widget that automatically displays an appropriate IconButton to show the `Drawer` when a Drawer is available in the `Scaffold`. The `Scaffold` automatically handles the edge-swipe gesture to show the `Drawer`.
+
+<!-- skip -->
+```Dart
+// Flutter
+@override
+Widget build(BuildContext context) {
+  return new Scaffold(
+    drawer: new Drawer(
+      child: new ListTile(
+        leading: new Icon(Icons.change_history),
+        title: new Text('Screen2'),
+        onTap: () {
+          Navigator.of(context).pushNamed("/b");
+        },
+      ),
+      elevation: 20.0,
+    ),
+    appBar: new AppBar(
+      title: new Text("Home"),
+    ),
+    body: new Container(),
+  );
+}
+```
+
+#### Preview
+
+|Android|iOS|
+|:---:|:--:|
+|<img src="/flutter-for-react-native/images/navigation_android.gif?raw=true" style="width:300px;" alt="Loading">|<img src="/flutter-for-react-native/images/navigation_iOS.gif?raw=true" style="width:300px;" alt="Loading">|
+
+<br>
+
+## Gesture detection and touch event handling
+
+To listen for and respond to gestures, Flutter includes taps, drags, and
+scaling. The gesture system in Flutter has two separate layers. The first layer
+includes raw pointer events, which describe the location and movement of
+pointers, (such as touches, mice, and styli movements), across the screen. The
+second layer includes gestures, which describe semantic actions that consist of
+one or more pointer movements.
+
+### How do I add a click or press listeners to a widget?
+
+In React Native, listeners are added to components using `PanResponder` or the `Touchable` components.
+
+<!-- skip -->
+```JavaScript
+// React Native
+<TouchableOpacity
+  onPress={() => {
+    console.log("Press");
+  }}
+  onLongPress={() => {
+    console.log("Long Press");
+  }}
+>
+  <Text>Tap or Long Press</Text>
+</TouchableOpacity>
+```
+
+For more complex gestures and combining several touches into a single gesture,
+ [`PanResponder`](https://facebook.github.io/react-native/docs/panresponder.html) is used.
+
+<!-- skip -->
+```JavaScript
+// React Native
+class App extends Component {
+
+  componentWillMount() {
+    this._panResponder = PanResponder.create({
+      onMoveShouldSetPanResponder:
```

Comment:
delete pre-defined

Reply:
This is another example - it looks like the comments and the text they refer to are not in sync.